apps/posterizarr/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM docker.io/library/alpine:3.21
+
+# Build arguments
+ARG TARGETARCH
+ARG VENDOR
+ARG VERSION=1.9.34
+
+# Set environment variables
+ENV UMASK="0002" \
+    TZ="Etc/UTC" \
+    DEBCONF_NONINTERACTIVE_SEEN="true" \
+    DEBIAN_FRONTEND="noninteractive" \
+    POWERSHELL_DISTRIBUTION_CHANNEL="PSDocker" \
+    VIRTUAL_ENV="/lsiopy"
+
+# Install APK dependencies
+RUN apk add --no-cache imagemagick imagemagick-jpeg imagemagick-heic libjpeg-turbo powershell tzdata
+
+# Set up PowerShell modules
+RUN echo "Setting up PowerShell modules..." && \
+    # Verify PowerShell installation
+    pwsh --version && \
+    # Set PSGallery to trusted and install module
+    pwsh -Command "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted; \
+                  Install-Module -Name FanartTvAPI -Scope AllUsers -Force; \
+                  if (-not (Get-Module -ListAvailable -Name FanartTvAPI)) { Write-Error 'FanartTvAPI module installation failed!'; exit 1 }; \
+                  Import-Module FanartTvAPI -Force; \
+                  if (-not (Get-Command -Name Add-FanartTvAPIKey -ErrorAction SilentlyContinue)) { Write-Error 'Add-FanartTvAPIKey function not found!'; exit 1 }; \
+                  \$modulePath = (Get-Module -ListAvailable -Name FanartTvAPI).Path; \
+                  \$moduleDir = Split-Path -Parent \$modulePath; \
+                  Write-Host \"Module directory: \$moduleDir\"" && \
+    # Make PowerShell modules accessible to nobody user
+    chmod -R 755 /usr/local/share/powershell && \
+    # Create config directory with correct permissions
+    mkdir -p /config && chmod 755 /config && \
+    echo "PowerShell modules setup completed successfully"
+
+# Download the repository
+ADD https://github.com/fscorrupt/Posterizarr/archive/refs/tags/${VERSION}.tar.gz /tmp/posterizarr.tar.gz
```

Comment:
Please do not use `ADD`, this adds extra weight to the Docker image as this layer does not get deleted. Instead use curl to add it directly in the `RUN`.